INSTALL.md

```diff
@@ -53,26 +54,26 @@ https://doc.ez.no/display/DEVELOPER/Step+1%3A+Installation
 
        ```bash
        $ curl -sS https://getcomposer.org/installer | php
-       $ php -d memory_limit=-1 composer.phar create-project --no-dev ezsystems/ezplatform
+       $ php -d memory_limit=-1 composer.phar create-project ezsystems/ezplatform ezplatform ^2.0
```

Comment:
`ezsystems/ezplatform` => `ezsystems/ezplatform-demo`

Reply:
Thanks @andrerom 
I think it makes more sense to keep the current install steps on the README in order to not confuse people. So I submitted a new PR without INSTALL.MD